plexapi/base.py

```diff
@@ -945,21 +1001,42 @@ class PlexSession(object):
     """
 
     def _loadData(self, data):
+        """ Load attribute values from Plex XML response. """
         self.live = utils.cast(bool, data.attrib.get('live', '0'))
-        self.player = self.findItem(data, etag='Player')
-        self.session = self.findItem(data, etag='Session')
         self.sessionKey = utils.cast(int, data.attrib.get('sessionKey'))
-        self.transcodeSession = self.findItem(data, etag='TranscodeSession')
 
         user = data.find('User')
         self._username = user.attrib.get('title')
         self._userId = utils.cast(int, user.attrib.get('id'))
 
         # For backwards compatibility
```

Comment:
the "# For backwards compatibility" should have been retained in property methods